src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -602,6 +615,17 @@ private void getCanvasResponse(HttpServletRequest request,
 		out.flush();
 	}
 
+	private void logAction(String uniqname, String originalUrl, String sessionId) {
```

Comment:
This could be more general (and shorter) if the arguments were originalUrl, valueTypeString, and valueValue (or such).  The caller will know the value and the string name of the value.  This could just format and log the string.

The name could be a bit clearer as well.